book.rb

```diff
@@ -0,0 +1,9 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title:, author:)
+    @title = title
+    @author = author
+    @rentals = []
+  end
```

Comment:
- [ ] Kinldy considers adding a method for adding rentals to achieve a `many-to-many` relation between `Person` and `Book`  using the intermediate class `Rental`. 

```ruby
 def add_rental(person, date)
    Rental.new(date, self, person)
  end
```